pycbc/inference/sampler_base.py

```diff
@@ -420,19 +461,33 @@ def write_acceptance_fraction(self, fp, max_iterations=None):
         """
         dataset_name = "acceptance_fraction"
         acf = self.acceptance_fraction
+
+        aa = start_iteration
```

Comment:
I'm going to strongly object to naming variables ``aa`` and ``bb``. I just watched Steven struggle with his work this week trying to decipher poor variable names elsewhere in PyCBC.